lib/request-options.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+module.exports = (logger, libOptions) => {
+    var requestOptions;
+
+    requestOptions = {};
+
+    /**
+     * @typedef {Object} options
+     * @property {Boolean} json Sets the body of the request to JSON.
+     * @proptery {Boolean} resolveWithFullResponse Get the full response
+     *                                             instead of just
+     *                                             the body.
+     * @property {Boolean} simple Get a rejection only if the request
+     *                            failed for technical reasons.
+     */
+
+    /**
+     * Creates and returns a requestOptions object with the Authorization
+     * header set with the authToken, json and resolveWithFullResponse set to
+     * true, simple set to false, and if the user set strictHostCheck to false,
+     * sets rejectUnauthorized to false.
```

Comment:
This documentation doesn't say anything that isn't already obvious from reading the code. Something along the lines of "Sets and returns the requestOptions object using the authToken." would be better. Even more succinct: "Preps options object for a Shelf request."